### PR TITLE
web update

### PR DESCRIPTION
**Pull request by yuvraajnarula**
Don't accept this pr until the website is done

### CODE REVIEW

**Review by ion05 — CHANGES_REQUESTED** [review]
Website reconstruction rejected

**Comment by ion05** [comment]
@yuvraajnarula  Remove [`0b58b1f` (#58)](https://github.com/life-sync/website2/pull/58/commits/0b58b1f14054c8c1da5cc9463e225de51bc0a0df) 
And decrease the width of the light blue section

File: index.html [hunk 1]
```diff
@@ -1,99 +1,131 @@
...
+  <div class="members">
+    <center>
+      <h1>Meet The Crew</h1>
+    </center> 
+    <br>
+    <section class="team">
+      <div class="card">
+        <img src="https://avatars0.githubusercontent.com/u/49789627?s=460&u=33bceea03dce0d02d2dd7d87b3ce0cac5ce1c42d&v=4" alt="Aayan Agarwal">
+        <h1>Aayan Agarwal</h1>
+        <div class="media">
```
Comment (ion05) [inline index.html:41]: Add title over here eg. Mobile App Developer

File: index.html [hunk 2]
```diff
@@ -1,99 +1,131 @@
...
+          <div class="media-list">
```
Comment (ion05) [inline index.html:42]: Dont put social links inside a box

**Review by ion05 — CHANGES_REQUESTED** [review]

**Comment by ion05** [comment]
Remove comments with code in them

**Comment by yuvraajnarula** [comment]
I have to work on that part of code but later so that's why it is commented

File: index.html [hunk 3]
```diff
@@ -42,32 +42,38 @@ <h1 id="about">About Us</h1>
...
-          <h1>0</h1>
-          <p>Clients :(</p>
-        </center>
-      </div>
-      <div class="rect">
-        <center>
-          <h1>5</h1>
-          <p>Projects</p>
-        </center>
+    <div class="middle">
```
Comment (ion05) [inline index.html:51]: Jo bhi hai yeh, hatade @yuvraajnarula

**Pull request merged**